src/ctia/schemas/graphql/helpers.clj

```diff
@@ -289,11 +308,14 @@
   [^String object-name
    ^String description
    fields]
-  (-> (GraphQLInputObjectType/newInputObject)
-      (.name object-name)
-      (.description description)
-      (add-input-fields fields)
-      .build))
+    (get-or-update-type-registry
+      default-type-registry
+      object-name
+      #(-> (GraphQLInputObjectType/newInputObject)
+           (.name ^String object-name)
+           (.description ^String description)
+           (add-input-fields fields)
+           .build)))
```

Comment:
Please temporarily permit the funny indentation, a forthcoming commit will wrap this entire return value in a function and restore some order.